### PR TITLE
Sponsorship page

### PR DESCRIPTION
**Pull request by wrightmartin**
localhost-8000-sponsorship

### CODE REVIEW

**Review by jacksonj04 — APPROVED** [review]
I approve these changes, 👍 

Reminder before deployment: We'll need to create the 2017 sponsorship content block on production.

File: ainow/templates/ainow/sponsorship.html [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+            <li>Your branding / logo / company url on all conference materials, conference website and promotional media</li>
+            <li>Social media announcement when sponsorship is confirmed</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <p class="sponsorship-benefits-callout">If you have ideas for additional sponsorship benefits that are not listed, we would be happy to discuss how we can accommodate those. We are also happy to discuss the creation of bespoke sponsorship packages. <a href="#">Just get in touch</a>.</p>
```
Comment (jacksonj04) [inline ainow/templates/ainow/sponsorship.html]: This should be a `mailto:` link to @Gemmamysoc

File: ainow/templates/ainow/sponsorship.html [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="homepage-section homepage-section--secondary">
+  <div class="container attendee-container">
+    <div class="row">
+      <div class="col-md-6">
+        <h2>Who attends TICTeC?</h2>
```
Comment (jacksonj04) [inline ainow/templates/ainow/sponsorship.html]: Missing an `id="attendees"` to allow the link from earlier to work.

**Review by jacksonj04 — DISMISSED** [review]
Just a couple of quick fixes spotted by @Gemmamysoc

File: ainow/templates/ainow/sponsorship.html (hunk 2 above)
Comment (wrightmartin) [reply ainow/templates/ainow/sponsorship.html]: ah damn I remember thinking when I put it in I'd forget it

File: ainow/templates/ainow/sponsorship.html (hunk 1 above)
Comment (wrightmartin) [reply ainow/templates/ainow/sponsorship.html]: yep - she gave me an address for it, adding now

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**